src/styles/index.css

```diff
@@ -0,0 +1,256 @@
+.card {
+  height: 408px;
+  max-width: 200px;
+  transition-duration: 0.6s;
```

Comment:
The transition duration for the card is set to 0.6s, but the hover effect transition duration is set to 0.3s (line 95). Consider making these consistent to ensure uniform animation effects.